mage_integrations/mage_integrations/sources/stripe/__init__.py

```diff
@@ -1246,5 +1312,5 @@ def new_request(self, method, url, params=None, headers=None):
             Context.print_counts()
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
Why do you update it to double quotes here?

Reply:
Modified on 028336556fca308242837d69d26186c17481362d